src/components/Alert.a11y.tsx

```diff
@@ -0,0 +1,70 @@
+import { mount  } from 'enzyme';
+import * as React from 'react';
+
+import Alert from './Alert';
+
+describe('Alert', () => {
+  it('type of `error` has no Pa11y violations', async () => {
+    const component = mount(
+      <Alert
+        type="error"
+      >
+        <p>Error</p>
+      </Alert>
+    );
+    const results = await pa11y(component, {
+      ignore: [
+        'region',
+      ],
+    });
+    expect(results).toHaveNoPa11yViolations();
+  });
+
+  xit('type of `info` has no Pa11y violations', async () => {
```

Comment:
Why are these tests disabled?

Reply:
I disabled these when troubleshooting the css issue below, and forgot to reenable them. I pushed a fix to enable them.